README.md

```diff
@@ -0,0 +1,257 @@
+# Intermittent Energy Importers
+
+Powers the analytics backend of [intermittent.energy/](https://intermittent.energy/), providing deep insights into global electricity markets and renewable energy integration.
+
+A high-performance Ruby application for collecting, processing, and analyzing real-time power grid data worldwide. Built on PostgreSQL with [TimescaleDB](https://github.com/timescale/timescaledb/) for lightning-fast time series processing.
+
+Follow [@IntermittentNRG](https://x.com/IntermittentNRG) for updates.
+
+## Data Sources
+
+🌎 **Americas**
+- 🇺🇸 ⚡ [CAISO (California Independent System Operator)](lib/caiso.rb)
+- 🇺🇸 🔋 [EIA (U.S. Energy Information Administration)](lib/eia.rb)
+- 🇺🇸 💡 [ERCOT (Electric Reliability Council of Texas)](lib/ercot.rb)
+- 🇺🇸 🔌 [NYISO (New York Independent System Operator)](lib/nyiso.rb)
```

Comment:
ERCOT and NYISO are also not in use. EIA offers same data, but data directly from the TSO is usually better quality. Maybe should revisit.